### ISSUE TITLE
Keyboard focus, Style issue

### ISSUE THREAD

**Issue by Machenblau** [nwg-piotr/nwg-drawer#153]
Hello,

i have added the following to my .css file, to get rid of the blue outline of the search bar:
```css
*{
  all:unset;
}
```
now almost everything looks fine, except for the math result box (still white border with black  background - see #147 ) and the keyboard focus.
i've tried the following selector, to style the keyboard focus:
```css
button:focus,
button:active,
button:selected,
button:hover {
  background-color: rgba(200,20,20,0.500000);/* values are a result of a calculation in home-manager */
}
```
is there a selector for this?

**Comment by nwg-piotr** [comment]
No, just for the label, as stated in #147. Closing as duplicate.

**Comment by Machenblau** [comment]
thank you for the quick response - the label is fine, and not a problem (it still fits my theme, somehow).

My problem is: when i select a application via the arrow keys, i cannot see what is selected (i can only guess based on the description):

![Image](https://github.com/user-attachments/assets/9fa52b40-fcc5-43e6-bab2-f51e3737e9f4)

**Comment by nwg-piotr** [comment]
Possibly your solution is too general? I would just set `border: none` to get rid of te search box border:

```css
entry {
    border: none;
}
```

**Comment by Machenblau** [comment]
i've found my issue: i had set attributes to entry and #files-box like this:
```css
entry,#files-box{
  background-color: rgba(0,0,0,0.800000);
  border: 0.1rem solid rgba(125,2,2,0.660000);
  padding:0.5rem;
}
```
adding a space like this:
```css
entry,
#files-box{
  background-color: rgba(0,0,0,0.800000);
  border: 0.1rem solid rgba(125,2,2,0.660000);
  padding:0.5rem;
}
```
solved my issue.

For others reading this thread: button:focus is the correct selector for keyboard focus, button:hover is the selector for mouse focus.

```css
button:focus,
button:hover {
  background-color: rgba(200,20,20,0.500000);/* values are a result of a calculation in home-manager */
}
```
